Makefile

```diff
@@ -0,0 +1,62 @@
+# if provided, FILTER is used as the --filter argument to `swift test`.
+ifdef FILTER
+	FILTERARG = --filter $(FILTER)
+else
+	FILTERARG =
+endif
+
+ifdef DOCSVERSION
```

Comment:
I think we can get rid of this - it's an artifact of how we used to generate the driver docs, and should be taken out of the driver's makefile too.